.circleci/config.yml

```diff
@@ -2,88 +2,61 @@ version: 2
 jobs:
   build:
     docker:
-      - image: osresearch/heads-ubuntu:16.04
+      - image: osresearch/musl-cross:38e52db
```

Comment:
Having the toolchain prebuilt is a nice shortcut for testing, but wouldn't we better have a clean checkout of up to date, mainstream docker images (ubuntu:18.04, ubuntu:latest, fedora, etc) so that we can trace build problems from the host toolchain changes from one build to the other and across different images, the default being ubuntu:18:04?

Consequently, from CircleCI at least, anyone wanting to build Heads themselves would simply have to subscribe to the project.

I'm not a CI expert, but the ideal down this path would be to have the same Heads codebase, and have environment variables specifying the host docker image to be used and the board config to build, so that one CircleCI/GitlabCI config can build for all hosts and boards config, requiring only different CI configurations based on the same code to test different hosts and boards.